aries-site/src/utils/analytics.js

```diff
@@ -4,3 +4,21 @@
 export const analytics = data => {
   // console.log('\x1B[1;44m ANALYTICS \x1B[0m', data);
 };
+
+export const GA_TRACKING_ID = process.env.NEXT_PUBLIC_GOOGLE_TAG_MANAGER_ID;
+
+// https://developers.google.com/analytics/devguides/collection/gtagjs/pages
+export const pageview = url => {
+  window.gtag('config', GA_TRACKING_ID, {
+    page_path: url,
+  });
+};
+
+// https://developers.google.com/analytics/devguides/collection/gtagjs/events
+export const event = ({ action, category, label, value }) => {
+  window.gtag('event', action, {
+    event_category: category,
+    event_label: label,
+    value,
+  });
+};
```

Comment:
Instead of hard coding calls to these inside _app.js and such you could simply look at the event in line 5 above (in the analytics callback) and send the appropriate ga event there. That's what that callback is intended for.